src/views/onboarding/components/OnBoardingProductStepDelegates.tsx

```diff
@@ -37,6 +46,55 @@ export function OnBoardingProductStepDelegates({ product, legal, forward, back }
 
   const allPeople = { ...people, LEGAL: legal };
 
+  const validateUserData = async (user: UserOnCreate, prefix: string) => {
```

Comment:
could we try to put this function to an external file in order to write once the logic to send the REST request and handle to response? (under the utils folder, we could a create an "api" folder in which we could put the userValidate.ts file

The shared function could receive as parameters onSuccess, onValidationError, onGenericError to handle the changes between the 2 usages